src/main/java/seedu/address/logic/Messages.java

```diff
@@ -21,7 +21,7 @@ public class Messages {
                 "Multiple values specified for the following single-valued field(s): ";
     public static final String MESSAGE_CONSTRAINTS_LENGTH =
             "length should not exceed the limit of 30 characters";
-    public static final String MESSAGE_CONSTRAINTS_ALPHANUMERIC_LENGTH = " names should be alphanumeric and not "
+    public static final String MESSAGE_CONSTRAINTS_ALPHANUMERIC_LENGTH = "names should be alphanumeric and not "
```

Comment:
Should be name instead of names. Suppose its names, it would indicate name1 + name2 + name3 should be in alphanumeric and not exceed 30 characters.